docs/types.md

```diff
@@ -3,7 +3,15 @@
 A `TimeZone` is an abstract type that represents information regarding a specific time zone. Typically you will create an instance of a `TimeZone` by passing in a zone name to the convenience constructor `TimeZone`:
 
 ```julia
-TimeZone("Europe/Warsaw")
+julia> TimeZone("Europe/Warsaw")
+Europe/Warsaw (UTC+1/UTC+2)
+```
+
+You can also create a `TimeZone` by using a `tz` string literal:
```

Comment:
I used the terminology adopted here: https://docs.julialang.org/en/latest/manual/metaprogramming.html#Non-Standard-String-Literals-1 but I don't have a strong opinion, will change following your suggestion.